WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -751,7 +751,7 @@ private extension OrderDetailsDataSource {
     /// Returns an `empty` array if we can't find the product associated with order item or if the `addOns` feature is disabled.
     ///
     private func filterAddOns(of item: AggregateOrderItem) -> [OrderItemAttribute] {
-        guard let product = products.first(where: { $0.productID == item.productOrVariationID }), showAddOns else {
+        guard let product = products.first(where: { $0.productID == item.productID }), showAddOns else {
```

Comment:
I changed this back to using `productOrVariationID` and the unit tests passed. Should we protect this with a unit test? Or is this temporary?

Reply:
Because of the nature of `OrderDetailsDataSource` I've found it's quite hard to add test to this logic :-/ :shame: on me